### PR TITLE
Add master script

### PR DESCRIPTION
**Pull request by HDThang**
Modify training script in the flowing tasks to run mlflow while training models:
-audio classification
-contrastive-image-text
-image classification
-image pretraining
-language modeling
-multiple choice
-semantic segmentation
-speech recognition
-summarization
-text classification
-token classification
-translation

### CODE REVIEW

File: examples/pytorch/language-modeling/.nfs00000000864964a600000492 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+gpt2 8
+roberta-base 8
+#### end list ####
```
Comment (huonglarne) [inline examples/pytorch/language-modeling/.nfs00000000864964a600000492:3]: what is this file? did you commit it by mistake?

File: examples/pytorch/text-classification/.nfs00000000865205a400000493 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+bert-base-uncased 64 1
+facebook/bart-base 16 3
+bigscience/bloom-560m 16 3
```
Comment (huonglarne) [inline examples/pytorch/text-classification/.nfs00000000865205a400000493:3]: what is this file? did you commit it by mistake?

File: examples/pytorch/contrastive-image-text/run_clip.py [hunk 3]
```diff
@@ -508,13 +515,19 @@ def filter_corrupt_images(examples):
...
         elif last_checkpoint is not None:
             checkpoint = last_checkpoint
-        train_result = trainer.train(resume_from_checkpoint=checkpoint)
-        trainer.save_model()
-        tokenizer.save_pretrained(training_args.output_dir)
-        image_processor.save_pretrained(training_args.output_dir)
-        trainer.log_metrics("train", train_result.metrics)
-        trainer.save_metrics("train", train_result.metrics)
-        trainer.save_state()
+        with mlflow_runner:
```
Comment (huonglarne) [inline examples/pytorch/contrastive-image-text/run_clip.py:518]: I think using ML flow like this makes the code a bit hard to read. You dont have to fix it now but maybe in the future, refactor this into a function called log_to_ml_flow or something

**Review by huonglarne — COMMENTED** [review]
I noticed some file with strange names
And some need refactoring

File: examples/pytorch/audio-classification/run_audio_classification.py [hunk 4]
```diff
@@ -382,18 +385,31 @@ def compute_metrics(eval_pred):
         tokenizer=feature_extractor,
     )
 
+    # mlflow initial
+    experiment_id = mlflow.create_experiment('translateion-{}'.format(model_args.model_name_or_path))
```
Comment (yeomni) [inline examples/pytorch/audio-classification/run_audio_classification.py:389]: According to what you've mentioned in your Daily Scrum, if what I think is right, we should be still able to use MLFlow Tracking just using Accelerate instance. Then do we not need to add all these repetitive MLFlow APIs for each task?

**Review by loctxmoreh — CHANGES_REQUESTED** [review]
There is a couple of issues with the use of `mlflow`:
* Update `env.yml` to include `mlflow` as a dependencies
* Or, consider making `mlflow` optional. If `mlflow` is not installed, the script should still run
* Automatically start mlflow server (if available), before running the train script 
* Using the environment variable `MLFLOW_TRACKING_URI` in bash script instead of this line in Python: `mlflow.set_tracking_uri()`

Also, please remove temp files that are accidentally committed, and be mindful when adding file to commit

**Review by thuc-moreh — APPROVED** [review]

**Comment by yeomni** [comment]
@HDThang If you have resolved all the commented issues, please merge this PR.

**Comment by yeomni** [comment]
Closed as deprecated

**Pull request closed without merging**